New log file opened at Mon May 18 11:37:27 CEST 2015 by 8104@ArifCerit
Taking next connection to: 57123
New client attempting to connect on port: 57124-57123
  Address = /172.20.155.142:57123
This client on port 57124 has uuid c493f8ea-27ab-47be-ac9a-eedf840e59b2 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-c493f8ea-27ab-47be-ac9a-eedf840e59b2
Taking next connection to: 57123
New client attempting to connect on port: 57125-57123
  Address = /172.20.155.142:57123
Handshake exception on socket: 57125
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57123
New client attempting to connect on port: 57126-57123
  Address = /172.20.155.142:57123
This client on port 57126 has uuid ea9e4a64-8d7e-49d9-8d55-523f11afbdf1 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-c493f8ea-27ab-47be-ac9a-eedf840e59b2, activator-ea9e4a64-8d7e-49d9-8d55-523f11afbdf1
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
New client attempting to connect on port: 57187-57123
  Address = /172.20.155.142:57123
Handshake exception on socket: 57187
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57123
New client attempting to connect on port: 57188-57123
  Address = /172.20.155.142:57123
This client on port 57188 has uuid edc9a943-1d9d-45ba-8ce5-0e985c8ba39e configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-ea9e4a64-8d7e-49d9-8d55-523f11afbdf1, play-fork-run-edc9a943-1d9d-45ba-8ce5-0e985c8ba39e
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
New client attempting to connect on port: 57232-57123
  Address = /172.20.155.142:57123
Handshake exception on socket: 57232
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57123
New client attempting to connect on port: 57233-57123
  Address = /172.20.155.142:57123
This client on port 57233 has uuid 43f8cf54-7266-4cf4-a28e-88b78327246c configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-ea9e4a64-8d7e-49d9-8d55-523f11afbdf1, play-fork-run-43f8cf54-7266-4cf4-a28e-88b78327246c
Taking next connection to: 57123
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57123
